src/apps/backend/modules/account/account-service.ts

```diff
@@ -22,4 +34,81 @@ export default class AccountService {
   ): Promise<Account> {
     return AccountReader.getAccountById(params.accountId);
   }
+
+  public static async createPasswordResetToken(
```

Comment:
Just like access token, should password reset token by separate module, ex: `password-reset-token`